### PR TITLE
avoid misaligned pointer deref in ktx code

### PR DESCRIPTION
**Pull request by sethalves**
- fix ktx related asan warning

https://highfidelity.fogbugz.com/f/cases/14670/ktx-related-asan-warning

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9552/app_13113_PR_6a55e67ce9cd01ed29825d14889a1d0d2ad902c6.apk

File: libraries/gpu/src/gpu/Texture_ktx.cpp [hunk 1]
```diff
@@ -77,10 +78,12 @@ struct GPUKTXPayload {
...
+
         // We can't copy the bitset in Texture::Usage in a crossplateform manner
         // So unserialize it manually
-        _usage = Texture::Usage(*(const uint32*)data);
+        uint32 usageData;
+        memcpy(&usageData, data, sizeof(uint32));
+        _usage = Texture::Usage(usageData);
         data += sizeof(uint32);
 
         _usageType = *(const TextureUsageType*)data;
```
Comment (kencooke) [inline libraries/gpu/src/gpu/Texture_ktx.cpp]: Seems like this is only aligned by happenstance. Should use memcpy() for consistency.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9552/HighFidelity-Beta-PR13113-6a55e67ce9cd01ed29825d14889a1d0d2ad902c6.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9552/HighFidelity-Beta-Interface-PR13113-6a55e67ce9cd01ed29825d14889a1d0d2ad902c6.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9555/app_13113_PR_677c99abd46e429090a4ebd3d85b2f7145e9ea6e.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9555/HighFidelity-Beta-PR13113-677c99abd46e429090a4ebd3d85b2f7145e9ea6e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9555/HighFidelity-Beta-Interface-PR13113-677c99abd46e429090a4ebd3d85b2f7145e9ea6e.exe

**Comment by kencooke** [comment]
The object code is unchanged, so no QA is needed. PASS.

**Pull request merged**